P5/Source/Guidelines/en/DI-PrintDictionaries.xml

```diff
@@ -2653,30 +2653,33 @@ sometimes informally stated values for the particular descriptive features, the
 of the content and of the container objects becomes relevant, in view of the growing
 tendency to interlink pieces of information across Internet resources. In such
 cases, it becomes crucial to be able to encode the fact that whether the information on,
-for instance, the value of the grammatical category of Number is provided as "sg.", "sing.", "Singular", or
-equivalently "poj." in Polish, or "Ez." in German, etc., what is actually referred to is always the same 
+for instance, the value of the grammatical category of Number is provided as <q>sg.</q>, <q>sing.</q>, <q>Singular</q>, or
+equivalently <q>poj.</q> in Polish, or <q>Ez.</q> in German, etc., what is actually referred to is always the same 
 grammatical value that can be rendered with a plethora of markers, depending on the publisher, language, or lexicographic tradition.
 In order to signal that this variety of surface markers in fact indicate the same
 underlying value, it is possible to align them with an external inventory of standardized
-values. The TEI provides means to align grammatical categories as well as their content
-with the ISOcat reference, which is a Web implementation of <ref target="#ISO-12620">ISO 12620</ref>.</p>
+values. The TEI provides the <ident type="class">att.datcat</ident> attribute class for the purpose of aligning grammatical (or indeed any sort of) categories as well as their values
```

Comment:
It is meant to be broad, because the need to fix references goes beyond linguistic applications (wouldn't FRBR apply? it's a bit hard for me to gauge the need, not being an expert in that field, but these attributes _are_ intended for general use, also where they complement `<equiv>`). I'd rather not put time into this in this round, if possible (I got behind with all the other deadlines), so I can delete the brackets, but then I'd be telling only half the story. Is there a chance to leave this as-is and expand afterwards, when I know more about other fields in this regard, or maybe an expert on that happens to float by, and we catch them and put them to work? ;-)
And `@ana` is different -- it's for analysis, while datcats are for fixing references, establishing identities or confirming non-identities (again, FRBR could be an analogue, I think).